src/napari/_vispy/overlays/zoom.py

```diff
@@ -0,0 +1,44 @@
+"""Vispy zoom box overlay."""
+
+from napari._vispy.overlays.base import ViewerOverlayMixin, VispyCanvasOverlay
+from napari._vispy.visuals.interaction_box import InteractionBox
+from napari.components.overlays.zoom import ZoomOverlay
+from napari.settings import get_settings
+
+
+class VispyZoomOverlay(ViewerOverlayMixin, VispyCanvasOverlay):
+    """Zoom box overlay.."""
+
+    def __init__(self, viewer, overlay: ZoomOverlay, parent=None):
+        super().__init__(
+            node=InteractionBox(),
+            viewer=viewer,
+            overlay=overlay,
+            parent=parent,
+        )
+
+        self.overlay.events.canvas_positions.connect(self._on_positions_change)
+
+        self._on_visible_change()
```

Comment:
@brisvag I'm wondering if it would be worth considering the option of a `__post_init__` hook to avoid having to call `__init__` where it's not necessary; looks to me that this overlay might be inherited directly from `VispySelectionOverlay`, or am I mistaken?